scripts/celo/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+FROM us.gcr.io/celo-testnet/celo-node:1.0.0
+
+WORKDIR /root
+
+COPY ./genesis.json .
+COPY ./key.txt .
+#COPY ./keystore /root/keystore

```

Comment:
Why is all the keystore stuff omitted?

Reply:
Was trying to debug the dockerfile and commented out that line, will uncomment